tests/content/entity/zone/create/test.md

```diff
@@ -1,16 +1,39 @@
 # Entity Zone Create
+## General
+This test will build the following zone hierarchy:
 
-### Preconditions
-In an empty region of a domain with editing rights
+![](./hierarchy.png) 
 
-### Steps
+The tests consist of setting zone component parameters and moving through the zones.
 
-#### Step 1
-- run this [script URL](./create.js?raw=true) (from menu/Edit/Open and Run scripts From URL...). 
+## Preconditions
+Run interface and delete all content
+
+## Steps
+
+### Step 1
+- run this [script URL](./create.js?raw=true) (from menu: Edit->Open and Run Script File...). 
 - expected ![](./create.png)
-  - A new zone entity called "test create zone" centered on my avatar position, a box of 5 meters side length
-  - The keylight color is turned to green and the background is activated and blue
-  - The world axis aligned bounding box of the zone is visible
-  - The zone stack tool is enabled and represent the newly created zone's key light (green) and background (blue) before the default zone graphics components
+  - A 1000m x 1000m flat terrain
+  - 4 areas are defined on the terrain.  Each area corresponds with a matching zone; the zones are all 10 m high.
+  
+### Step 2
```

Comment:
I would suggest that this step is the test you put in the Hierarchy  test case
You would need to replay the part of the script that create the zone layout you could duplicate that code